patterns/templates/patterns-sailfish-device-configuration-@DEVICE@.inc

```diff
@@ -11,7 +11,7 @@ Requires: patterns-sailfish-rnd
 # dev-tools pattern will be fixed in the next release
```

Comment:
Is this comment still needed?

Reply:
Yes, because this repo is facing community, who are using 3.4.0.